### PR TITLE
08_pytorch_paper_replicating.ipynb

### PR DESCRIPTION
**Pull request by pritesh2000**
08: pytorch paper replicating
https://www.learnpytorch.io/08_pytorch_paper_replicating/

inlcuding -> including
all of the other model's -> all of the other models
Similar to a how a convolutional neural network (CNN) -> Similar to how a convolutional neural network (CNN)

Appendex -> Appendix
favoured -> preferred
compute -> computation

indexs -> indexes
refering -> referring
convoluational -> convolutional
Flattned -> Flattened
preprended -> prepended
embeddding -> embedding
noramlize -> normalize

wouldn't necessary help -> wouldn't necessarily help
this on it's own -> this on its own

architeture -> architecture
defintion -> definition

throguh -> through
throguh -> through

architecure -> architecture
its been tested -> it's been tested 
divisble -> divisible
severly -> severely

agonistc -> agnostic
creating in 07. ->  created in 07.
form -> from
mimicing -> mimicking
thinks its pizza -> thinks it's pizza
similiar -> similar
To do we, let's can use the -> To do so, let's use the

### CODE REVIEW

**Comment by mrdbourke** [comment]
Thank you! I really appreciate all of these :)
